webapp/src/client/client.ts

```diff
@@ -1,40 +1,45 @@
 import {Client4} from 'mattermost-redux/client';
 import {ClientError} from 'mattermost-redux/client/client4';
 
+import {Item, TooltipData} from 'src/types/gitlab_items';
+import {APIError, ConnectedData, GitlabUsersData, LHSData, SubscriptionData} from 'src/types';
+
 export default class Client {
-    setServerRoute = (url) => {
+    private url = '';
+
+    setServerRoute(url: string): void {
         this.url = `${url}/api/v1`;
     }
 
-    getConnected = async (reminder = false) => {
+    async getConnected(reminder: boolean): Promise<ConnectedData> {
         return this.doGet(`${this.url}/connected?reminder=` + reminder);
```

Comment:
What do you think about doing this pattern instead?

```suggestion
    async getConnected(reminder: boolean) {
        return this.doGet<ConnectedData>(`${this.url}/connected?reminder=` + reminder);
```

Code for reference
- https://github.com/mattermost/mattermost-plugin-github/blob/ef7608ed151ed5defc1a2ac98a36bb5b388fa7c2/webapp/src/client/client.ts#L32
- https://github.com/mattermost/mattermost-plugin-github/blob/ef7608ed151ed5defc1a2ac98a36bb5b388fa7c2/webapp/src/client/client.ts#L79